SPECS/cmake/cmake.spec

```diff
@@ -1,7 +1,7 @@
 %global major_version 3
 Summary:        Cmake
 Name:           cmake
-Version:        3.29.6
+Version:        3.30.3
```

Comment:
This is a toolchain package, so you'll need to update the toolchain manifests as well (see: PR check failure messages).